example/transforms/xccdf2table-profilecisrefs.xslt

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
```

Comment:
Is think  this file is not used because no tables are built.
This should be removed or a CIS reference table added.